### PR TITLE
Docs: add correlations content to What's new in Cloud

### PR DESCRIPTION
**Pull request by imatwawana**
Add Correlations content to What's new in Cloud

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 1]
```diff
@@ -18,6 +18,34 @@ weight: -37
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
 
+## Correlations editor in Explore
+
+<!-- Kristina Durivage -->
+<!-- OSS, Enterprise -->
+<!-- already in on-prem -->
+
+<Date>
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:27]: Looking for an approximate release date here

File: docs/sources/whatsnew/whats-new-next/index.md [hunk 2]
```diff
@@ -18,6 +18,34 @@ weight: -37
...
+
+To enable this feature, contact Grafana Support.
+
+## Create correlations for provisioned data sources
+
+<!-- Piotr Jamróz -->
+<!-- OSS, Enterprise -->
+<!-- already in on-prem -->
+
+<Date>
```
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:41]: Looking for an approximate release date here

File: docs/sources/whatsnew/whats-new-next/index.md (hunk 1 above)
Comment (ifrost) [reply docs/sources/whatsnew/whats-new-next/index.md:27]: It was made available for fast channel on 2023/10/03.

File: docs/sources/whatsnew/whats-new-next/index.md (hunk 2 above)
Comment (ifrost) [reply docs/sources/whatsnew/whats-new-next/index.md:41]: It was made available for fast channel on 2023/09/13.

File: docs/sources/whatsnew/whats-new-next/index.md (hunk 1 above)
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:27]:
```suggestion
October 3, 2023
```

File: docs/sources/whatsnew/whats-new-next/index.md (hunk 2 above)
Comment (imatwawana) [inline docs/sources/whatsnew/whats-new-next/index.md:41]:
```suggestion
September 13, 2023
```

**Comment by imatwawana** [comment]
@ifrost - If this looks good to you, can you give an approval?

**Review by Eve832 — APPROVED** [review]

**Review by ifrost — APPROVED** [review]
Looks great 🙌 Thanks @imatwawana!

**Comment by ifrost** [comment]
@imatwawana I've just merged on-prem what's new update https://github.com/grafana/grafana/pull/76505

**Pull request merged**